pkg/sdk/testint/pipes_integration_test.go

```diff
@@ -402,4 +402,15 @@ func TestInt_PipesShowByID(t *testing.T) {
 		require.NoError(t, err)
 		require.Equal(t, id2, e2.ID())
 	})
+
+	t.Run("show by id: check fields", func(t *testing.T) {
```

Comment:
can be added to assertPipe, no need for a separate test

Reply:
Done.